tests/test_primary.py

```diff
@@ -14,22 +14,23 @@
 
 import uptane # Import before TUF modules; may change tuf.conf values.
 
-from six.moves.urllib.error import URLError
-
-import tuf
-import tuf.formats
-import tuf.client.updater # to test one of the fields in the Primary object
-import tuf.keys # to validate a signature
-
 import unittest
 import os.path
 import time
 import copy
 import shutil
 import hashlib
+
+from six.moves.urllib.error import URLError
```

Comment:
Was this an organizational change?  Did you want to organize the stdlib modules first, then libraries, etc.?